cassandra/cassandra-statefulset.yaml

```diff
@@ -1,10 +1,15 @@
-apiVersion: "apps/v1beta1"
+apiVersion: "apps/v1" # for versions before 1.9.0 use apps/v1beta2  and before 1.8.0 use apps/v1beta1
 kind: StatefulSet
 metadata:
   name: cassandra
+  labels:
```

Comment:
I think adding this label is not necessary as part of apps/v1 migration. Do other PRs made to other Kubernetes repos also add this label? If not, let’s not add labels.

Reply:
It is recommended to add this lable in kubernetes.io examples  links： https://kubernetes.io/docs/tutorials/stateful-application/cassandra/

Reply:
That's fair, it should be here in the first place.